TAF/testScenarios/integrationTest/UC_device_service/device_configuration.robot

```diff
@@ -62,34 +63,37 @@ Config004 - Verfiy reading contains units when ReadingUnits is false
 
 Config005 - Verify OperationalState when AllowedFails is default with failed device requests
 # Default AllowedFails is 0(set to zero to disable automatic disablement of devices)
-    [Tags]  skipped
-    Given Create Device For device-modbus With Invalid Port
-    When Set specified device write command
-    Then Device OperationalState Should Be Up
-    [Teardown]  Delete Device By Name
+    Set Test Variable  ${device_name}  Modbus-Test-Device
+    ${set_data}  Create Dictionary  Modbus_DeviceValue_INT16_RW=60
+    Given Create Unavailable Modbus device
```

Comment:
For consistency, slightly adjust the order of the keywords. Please also update the other test cases accordingly.
```suggestion
    ${set_data}  Create Dictionary  Modbus_DeviceValue_INT16_RW=60
    Given Set Test Variable  ${device_name}  Modbus-Test-Device
    And Create Unavailable Modbus device
```